src/app/views/PersonDetail.svelte

```diff
@@ -73,44 +121,63 @@
        background-size: cover;
        background-image: linear-gradient(to bottom, ${rgba}, ${rgba}, ${rgb}), url('${banner}')`} />
 
-<div class="flex gap-4 text-neutral-100">
-  <PersonCircle {pubkey} class="mt-1 h-12 w-12 sm:h-32 sm:w-32" />
-  <div class="flex min-w-0 flex-grow flex-col gap-4">
-    <div class="flex flex-col">
-      <div class="flex w-full items-center justify-between gap-4">
-        <PersonName class="w-full text-2xl" {pubkey} displayNpubCopyButton />
-        <div class="hidden xs:block">
-          <PersonActions {pubkey} />
+<div>
+  <div class="relative flex gap-4 bg-tinted-700 p-6 text-neutral-100">
+    <div>
+      <PersonCircle {pubkey} class="mt-1 h-12 w-12 sm:h-32 sm:w-32" />
+      {#each actions as action (action.label)}
+        <div animate:flip={{delay: 200, duration: 1000, easing: bounceInOut}}>
```

Comment:
This animation is really funky. Maybe use `transition:slide` instead. Also, there's no need to iterate over actions here, just use conditionals, the extra complexity isn't worth it.

https://github.com/user-attachments/assets/6f6a78f9-91be-4689-85e6-e8717386ddd9

Reply:
yes the code is clunky, the only reason to use a keyed each here is to have button message gently take the empty space from the disappearing follow button by sliding in.
Not sure I'll get this same behaviour with transition:slide

Reply:
it's true that the animation does not look that good anyway